diffs/heartbeat.diff

```diff
@@ -0,0 +1,1933 @@
++               Configuring Heartbeat
++              </a>
++              for a full description of each configuration option.
++             </p>
+             </li>
+             <li class="listitem">
+              <p class="simpara">
+@@ -551,17 +551,17 @@
+                      </a>
+                      or equivalent privileges.
+                     </p>
+-                    <p>
+-                     For more information, see
+-                     <a class="xref" href="securing-heartbeat.html" title="Securing Heartbeat">
+-                      Securing Heartbeat
+-                     </a>
+-                     .
+-                    </p>
+                    </td>
+                   </tr>
+                  </table>
+                 </div>
++                <p class="simpara">
++                 For more information, see
++                 <a class="xref" href="securing-heartbeat.html" title="Securing Heartbeat">
++                  Securing Heartbeat
++                 </a>
++                 .
++                </p>
+                </li>
+               </ul>
+              </div>
+html/en/beats/heartbeat/master/heartbeat-geoip.html
+--- /dev/fd/63
++++ html/en/beats/heartbeat/master/heartbeat-geoip.html
+@@ -231,8 +231,8 @@
+               </code>
+               processors to add location information to the event. For example, you can use the Console in Kibana to create the following pipeline:
+              </p>
+-             <div class="pre_wrapper lang-json">
+-              <pre class="programlisting prettyprint lang-json">PUT _ingest/pipeline/geoip-info
++             <div class="pre_wrapper lang-console">
++              <pre class="programlisting prettyprint lang-console">PUT _ingest/pipeline/geoip-info
+ {
+   "description": "Add geoip info",
+   "processors": [
+html/en/beats/heartbeat/master/heartbeat-getting-started.html
+html/en/beats/heartbeat/master/heartbeat-installation.html
+html/en/beats/heartbeat/master/heartbeat-overview.html
+html/en/beats/heartbeat/master/heartbeat-reference-yml.html
+html/en/beats/heartbeat/master/heartbeat-starting.html
+html/en/beats/heartbeat/master/heartbeat-template.html
+html/en/beats/heartbeat/master/http-endpoint.html
+html/en/beats/heartbeat/master/ilm.html
+--- /dev/fd/63
++++ html/en/beats/heartbeat/master/ilm.html
+@@ -283,7 +283,7 @@
+           <p>
+            The index lifecycle write alias name. The default is
+            <code class="literal">
+-            heartbeat-{agent.version}
++            heartbeat-%{agent.version}
+            </code>
+            . Setting this option changes the alias name.
+           </p>
+html/en/beats/heartbeat/master/include-fields.html
+html/en/beats/heartbeat/master/index.html
+--- /dev/fd/63
++++ html/en/beats/heartbeat/master/index.html
+@@ -341,7 +341,7 @@
+               <li>
+                <span class="chapter">
+                 <a href="configuration-heartbeat-options.html">
+-                 Set up Heartbeat monitors

```

Comment:
Fixed in https://github.com/elastic/beats/commit/ebb0e742cf0953397da473e5d9a931e1b55a2a4c